src/main/java/pro/cloudnode/smp/bankaccounts/BankAccounts.java

```diff
@@ -79,6 +81,13 @@ public void onEnable() {
                 new GUI()
         };
         for (final @NotNull Listener event : events) getServer().getPluginManager().registerEvents(event, this);
+
+        // Setup PlaceholderAPI Integration
+        if(Bukkit.getPluginManager().getPlugin("PlaceholderAPI") != null) {
+            new PAPIIntegration().register();
+        } else {
+            getLogger().log(Level.WARNING, "PlaceholderAPI not found. Placeholders will not work.");
```

Comment:
I think maybe we can change this to INFO/LOG "PlaceholderAPI not available. Skipping integration." so people don't confuse this for our mini message placeholders being unavailable

Reply:
Sure will improve upon know I get home